nf_core/modules/module_test.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+"""
+The ModulesTest class runs the tests locally
+"""
+
+import logging
+import questionary
+import os
+import pytest
+import sys
+import rich
+
+import nf_core.utils
+from .modules_repo import ModulesRepo
+
+log = logging.getLogger(__name__)
+
+
+class ModulesTest(object):
+    """
+    Class to run module pytests.
+
+    ...
+
+    Attributes
+    ----------
+    module_name : str
+        name of the tool to run tests for
+    no_prompts : bool
+        flat indicating if prompts are used
+    pytest_args : tuple
+        additional arguments passed to pytest command
+
+    Methods
+    -------
+    run():
+        Run test steps
+    __check_inputs():
```

Comment:
A single underline would probably be preferable. Indicating that these methods are private is probably a big enough hurdle. So `_check_inputs()` instead of `__check_inputs()` and the same for the other two "private" methods. Sorry for not being more specific in the first place.

Reply:
Since you made them dunder methods, I wonder if you had a good reason?